### PR TITLE
Use more generic message when asking the user to authenticate

### PR DESCRIPTION
**Pull request by TheOneRing**
image

Fixes: #9896

### CODE REVIEW

File: src/gui/askforoauthlogindialog.cpp [hunk 1]
```diff
@@ -28,7 +28,7 @@ AskForOAuthLoginDialog::AskForOAuthLoginDialog(AccountPtr accountPtr, QWidget *p
 {
     _ui->setupUi(this);
 
-    _ui->label->setText(tr("The account %1 has been logged out by the server.\n\nPlease use your browser to log in to %2.").arg(accountPtr->displayName(), Theme::instance()->appNameGUI()));
+    _ui->label->setText(tr("The account %1 is currently logged out.\n\nProceed to authenticate using your browser %2.").arg(accountPtr->displayName(), Theme::instance()->appNameGUI()));
```
Comment (fmoc) [inline src/gui/askforoauthlogindialog.cpp:31]: Grammatically this makes no sense. Maybe `Proceed to authenticate to %2 using your browser.`? But then again, what's the advantage over the old message?

**Review by fmoc — CHANGES_REQUESTED** [review]

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12540/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12540/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12540/guiReportUpload/serverlog.log

Screenshots:
- [File_with_spaces_in_the_name_can_sync.png](https://cache.owncloud.com/public/owncloud/client/12540/guiReportUpload/screenshots/File_with_spaces_in_the_name_can_sync.png)
- [logging_out.png](https://cache.owncloud.com/public/owncloud/client/12540/guiReportUpload/screenshots/logging_out.png)
- [simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12540/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12540/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png)

</details>

File: src/gui/askforoauthlogindialog.cpp [hunk 2]
```diff
@@ -28,7 +28,7 @@ AskForOAuthLoginDialog::AskForOAuthLoginDialog(AccountPtr accountPtr, QWidget *p
 {
     _ui->setupUi(this);
 
-    _ui->label->setText(tr("The account %1 has been logged out by the server.\n\nPlease use your browser to log in to %2.").arg(accountPtr->displayName(), Theme::instance()->appNameGUI()));
+    _ui->label->setText(tr("The account %1 is currently logged out.\n\nProceed to authenticate using your browser.").arg(accountPtr->displayName()));
```
Comment (fmoc) [inline src/gui/askforoauthlogindialog.cpp:31]: I think some "Please [...]" is missing in that sentence. I guess "Please authenticate using your browser" would do.

**Review by fmoc — APPROVED** [review]
Feel free to merge, or further improve the message.

**Pull request merged**